set.mm

```diff
@@ -674653,6 +674654,24 @@ is in the span of P(i)(X), so there is an R-linear combination of
 $( (End of Stefan O'Rear's mathbox.) $)
 $( End $[ set-mbox-so.mm $] $)
 
+$( Begin $[ set-mbox-np.mm $] $)
+$( Skip $[ set-main.mm $] $)
+$(
+#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#
+  Mathbox for Noam Pasman
+#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#
+$)
+
+  ${
+    r1sssucd.1 $e |- ( ph -> A e. On ) $.
+    $( Deductive form of ~ r1sssuc.  (Contributed by Noam Pasman,
```

Comment:
A link (~ risssuc) must not be followed by any character other than a blank or a line feed.  This error was detected by the automatic checks.

Reply:
Looks like the revision fixed this and the automated checks are passing.

@wlammen needs to remove the "request changes" for this to be mergeable, but I suppose in general it isn't a bad idea to wait a short time before merging in case there are further comments (not that we have a fixed time period).